addons/wowl/static/src/legacy_prod_infra/legacy_setup.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, config, utils } from "@odoo/owl";
```

Comment:
Split of the two files to avoid mixing production execution and testing execution

Reply:
in particular, this file has side effects, whereas legacy.ts only exports stuff